marlowe-dashboard-client/src/Contract/State.purs

```diff
@@ -84,9 +84,9 @@ dummyState =
   emptyMarloweState = Semantic.State { accounts: mempty, choices: mempty, boundValues: mempty, minSlot: zero }
 
 mkInitialState :: WalletDetails -> Slot -> PlutusAppId -> ContractHistory -> Maybe State
-mkInitialState walletDetails currentSlot followerAppId contractHistory = case contractHistory of
-  None -> Nothing
-  History marloweParams marloweData transactionInputs ->
+mkInitialState walletDetails currentSlot followerAppId { chParams, chHistory } = case chParams of
```

Comment:
chParams >>= \(marloweParams /\ marloweData)

Reply:
Good point. I think I prefer `bind` here, though I'm not quite sure why.